sdks/java/io/rrio/src/main/java/org/apache/beam/io/requestresponse/RedisClient.java

```diff
@@ -61,6 +62,15 @@ long decr(String key) throws UserCodeExecutionException {
     }
   }
 
+  /** Get a byte array associated with a byte array key. Returns null if key does not exist. */
+  byte @Nullable [] getBytes(byte[] key) throws UserCodeExecutionException {
```

Comment:
It's actually import to return a nullable byte array instead of an empty one as this follows the underlying Redis client. The checker complained with what I personally think is the more readable `@Nullable byte[]` and for some reason preferred `byte @Nullable []`.

Reply:
gotcha thanks for explaining!